spec/controllers/educators_controller_spec.rb

```diff
@@ -215,4 +215,21 @@ def make_request
     end
 
   end
+
+  describe '#notes_feed' do
+    def make_request
+      request.env['HTTPS'] = 'on'
+      get :notes_feed
+    end
+
+    context 'educator with homeroom' do
```

Comment:
Maybe some authentication specs and test the expected outcome of the JSON endpoint?